corehq/form_processor/models.py

```diff
@@ -1216,6 +1216,14 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges):
     last_modified = models.DateTimeField(auto_now=True)
 
     @property
+    def last_modified_date(self):
+        return self.last_modified
+
+    @property
+    def product_id(self):
+        return self.entry_id
```

Comment:
Although also pretty confusing if the code sometimes calls it one and sometimes calls it the other—someone seeing that would assume there's some subtle difference